docs/serving/samples/hello-world/helloworld-csharp/helloworld-csharp.csproj

```diff
@@ -1,15 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+    <RootNamespace>helloworld_csharp</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Folder Include="wwwroot\" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />

```

Comment:
The NuGet package `Microsoft.AspNetCore.Razor.Design` is not referenced by any source code in the sample app. Therefore, this dependency should not be added.

Reply:
Changes in the C# project come from the 2.2 SDK. I followed the [tutorial](https://knative.dev/docs/serving/samples/hello-world/helloworld-csharp/). If you create a new empty web project (`dotnet new web -o helloworld-csharp`), you will get this.

Nonetheless, I shouldn't have included these changes in this PR. For the sake of simplicity, I'm going to remove the whole C# part.

Reply:
Interesting. I'll have to play around with the recent `dotner new` commands. So I guess the question becomes should the sample be the smallest code that works, or should it be something that most people will end up generating by using popular tools.